pages/dkp/kaptain/2.0.0/release-notes/2.0.0/index.md

```diff
@@ -57,7 +57,46 @@ Kaptain 2.0 drastically reduces the bundle sizes improving download times, espec
 - Kubeflow Pipelines 1.8.1
 - Training Operator 1.4.0
 - Tensorflow 2.8.0
-- Pytorch 1.11.0
+- PyTorch 1.11.0
 - CUDA 11.4
 - MXNet 1.9
 - Horovod 0.24.2
+
+## Known issues
```

Comment:
should this be here or in the release notes for 2.1.0?
(I assume that it should be here, but just double confirming)

Reply:
We are trying to work on a solution for Kap 2.1, so we are not documenting the workaround for 2.1 (yet).

Reply:
yeah that makes sense. happy with this remaining in docs for 2.0.0 👌